src/data_structures/message.cpp

```diff
@@ -7,19 +7,19 @@
 #include "../../include/data_structures/message.hpp"
 
 /***************************************************/
-/************* Output stream ************************/
+/************* Output stream ***********************/
 /***************************************************/
 
-ostream& operator<<(ostream& os, const Message_t& msg) {
-  os << msg.value;
-  return os;
+ostream& operator<<(ostream& os, const message_t& msg) {
+    os << msg.value;
+    return os;
 }
 
 /***************************************************/
 /************* Input stream ************************/
 /***************************************************/
 
-istream& operator>> (istream& is, Message_t& msg) {
-  is >> msg.value;

```

Comment:
Similarly "is" could be replaced with "input_stream".The variable names should be self-explanatory.

Reply:
Many thanks for this feedback. It is resolved now.